src/components/ProjectField/InvertedCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+
+import { ProjectFieldProps, renderFieldValue } from '.';
+import GridEntryWrapper from './GridEntryWrapper';
+
+const InvertedCard = ({ label, md, value, backgroundColor }: ProjectFieldProps) => {
+  const theme = useTheme();
+
+  return (
+    <GridEntryWrapper md={md || 4} sx={{ marginBottom: 0 }}>
+      <Box
+        borderRadius={theme.spacing(1)}
+        textAlign={'left'}
+        margin={`0 ${theme.spacing(2)}`}
+        padding={`${theme.spacing(2)} ${theme.spacing(2)}`}
+        sx={{
+          backgroundColor: backgroundColor,
+        }}
+      >
+        <Grid container alignContent={'left'}>
+          <Grid item xs={12}>
+            {renderFieldValue(value)}
+            <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={600} marginBottom={theme.spacing(1)}>
```

Comment:
Do this also need space/margin in the top?

Reply:
Actually now that I look at the design, I think it needs _only_ a marginTop and no marginBottom haha